l10n_it_fatturapa_in/models/account.py

```diff
@@ -15,6 +16,35 @@ class AccountInvoice(models.Model):
         "einvoice.line", "invoice_id", string="Lines Detail",
         readonly=True, copy=False)
 
+    fatturapa_amount_untaxed = fields.Monetary(
```

Comment:
@robyf70 è possibile usare `e_invoice` invece di `fatturapa`? (Se non l'hai già installata in produzione)

Reply:
@eLBati Si certo! Non ho l'ancora installata in produzione, mi interessava fare un giro di review qui ;)